docker-bake.hcl

```diff
@@ -26,6 +26,24 @@ target "_common" {
     }
 }
 
+target "_platforms" {
+    platforms = [
+        "darwin/amd64",
+        "darwin/arm64",
+        "linux/386",
+        "linux/amd64",
+        "linux/arm/v6",
+        "linux/arm/v7",
+        "linux/arm64",
+        "linux/ppc64le",
+        "linux/riscv64",
+        "linux/s390x",
+        "windows/386",
```

Comment:
Looks like the `cross` targets are definitely taking the longest amount of time in CI, so perhaps we should remove them (at least from the default) 🤔 

(Not sure how much removing these saves us, but at least it would be _some_ 😅)

Reply:
Removed. Previous build ~13m 42s. Let's see how it goes now.

Reply:
~13m 46s, pretty much the same.

Reply:
LOL, well, it was worth trying 😁